tests/logger_tests.py

```diff
@@ -741,6 +747,9 @@ async def a_function(aaa):
     messages = [log["message"] for log in std_out if "message" in log]
     assert messages == ["amiddle"]
     assert std_out[-1]["log_info"]["func"] == "test_async_context"
+    if log_result:
+        logged_result = [log["action_result"] for log in std_out if "action_result" in log]
```

Comment:
would be good to check 'action_result' not in log when log_result=False

Reply:
good point, will add this check